lib/build.js

```diff
@@ -89,12 +89,18 @@ function sassInclude (filePath) {
 
 function sassKitFrontendDependency () {
   const timer = startPerformanceTimer()
+
   const internalGovUkFrontendDir = getInternalGovukFrontendDir()
   const internalGovUkFrontendConfig = fse.readJsonSync(path.join(internalGovUkFrontendDir, 'govuk-prototype-kit.config.json'))
-  const fileContents = internalGovUkFrontendConfig.sass
+
+  const govukFrontendSass = (internalGovUkFrontendConfig.sass || [])
```

Comment:
I'll do another push to remove the `|| []` as the internal version always has `sass`

```suggestion
  const govukFrontendSass = internalGovUkFrontendConfig.sass
```

Reply:
Good spot. Thanks.